modules/phase_field/src/materials/LinearIsoElasticPFDamage.C

```diff
@@ -0,0 +1,114 @@
+#include "LinearIsoElasticPFDamage.h"
+
+template<>
+InputParameters validParams<LinearIsoElasticPFDamage>()
+{
+  InputParameters params = validParams<LinearElasticMaterial>();
+  params.addRequiredCoupledVar("c","Order parameter for damage");
+  params.addParam<Real>("kdamage",1e-6,"Stiffness of damaged matrix");
+  return params;
+}
+
+LinearIsoElasticPFDamage::LinearIsoElasticPFDamage(const std::string & name,
+                                             InputParameters parameters) :
+  LinearElasticMaterial(name, parameters),
+  _c(coupledValue("c")),
+  _kdamage(getParam<Real>("kdamage")),
+  _dstress_dc(declareProperty<RankTwoTensor>("dstress_dc")),
+  _G0_pos(declareProperty<Real>("G0_pos")),
+  _dG0_pos_dstrain(declareProperty<RankTwoTensor>("dG0_pos_dstrain"))
```

Comment:
It would be a good idea to initialize _scaling here too just in case it's used before getScalingFactor() is called.